tests/testthat/test-preprocessing.R

```diff
@@ -75,5 +75,34 @@ test_succeeds("images arrays can be resized", {
   }
 })
 
+test_succeeds("flow images from dataframe works", {
+  
+  if (!reticulate::py_module_available("pandas"))
```

Comment:
If the function itself requires pandas we should note this in the documentation (and also do a check during the call to print a clear error message to that effect). Note that pandas will generally not be installed in TF environments installed via `install_tensorflow()` without special extra effort.

Reply:
Thanks JJ! I added a note to the docs here: https://github.com/rstudio/keras/pull/725/files#diff-151c30ec9c406dc7d593b4eeca5113a3R890 and I'll catch the error to print a clear message here: https://github.com/rstudio/keras/pull/725/files#diff-151c30ec9c406dc7d593b4eeca5113a3R910